src/main/scala/rocket/nbdcache.scala

```diff
@@ -6,11 +6,10 @@ import Chisel._
 import uncore.tilelink._
 import uncore.coherence._
 import uncore.agents._
-import uncore.util._
 import uncore.constants._
-import util.{ParameterizedBundle, DecoupledHelper}
+import uncore.util._
```

Comment:
I think if it's just used in the L1 caches it doesn't really need to be promoted to the global utils.